### PR TITLE
[#3530] Password requirements should be displayed when password requirements is enabled

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/3530 - [studio-ui] Password requirements should be displayed when password requirements is enabled

### CODE REVIEW

File: static-assets/scripts/main.js [hunk 1]
```diff
@@ -747,34 +750,66 @@
...
+            ['']
+          )[0].match(/{(.*?)}/) ||
+          ['', '']
+        )
+      )[1].split(',')[1];
+
+      const minLength = (
+        (
+          (
+            generalRegExp.match(/\(\?<minMaxLength>(.*){(.*?)}\)/) ||
```
Comment (rart) [inline static-assets/scripts/main.js:765]: wrong key

File: static-assets/scripts/main.js [hunk 2]
```diff
@@ -747,34 +750,66 @@
...
+            ['']
+          )[0].match(/{(.*?)}/) ||
+          ['', '']
+        )
+      )[1].split(',')[0];
+
+      const maxLength = (
+        (
+          (
+            generalRegExp.match(/\(\?<minMaxLength>(.*){(.*?)}\)/) ||
```
Comment (rart) [inline static-assets/scripts/main.js:775]: wrong key

File: static-assets/scripts/main.js [hunk 3]
```diff
@@ -747,34 +750,66 @@
...
+        )
+      )[1].split(',')[1];
+
       const messages = {
         hasNumbers: formatMessage(passwordRequirementMessages.hasNumbers),
         hasLowercase: formatMessage(passwordRequirementMessages.hasLowercase),
         hasUppercase: formatMessage(passwordRequirementMessages.hasUppercase),
         hasSpecialChars: formatMessage(passwordRequirementMessages.hasSpecialChars, { chars: (allowedChars ? `(${allowedChars})` : '') }),
         noSpaces: formatMessage(passwordRequirementMessages.noSpaces),
         minLength: formatMessage(passwordRequirementMessages.minLength, { min }),
```
Comment (rart) [inline static-assets/scripts/main.js:788]: you renamed var min above so it no longer exists. 
bear in mind the min message prop needs to be called min and the minmax is called minLength

Comment (rart) [reply static-assets/scripts/main.js:788]: create separate vars and regex processing for each. you can’t mix to get min and max separately as they are not guaranteed to be together

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/scripts/main.js (hunk 3 above)
Comment (vestrada) [reply static-assets/scripts/main.js:788]: Hi Roy, 
I'm not sure if I'm understanding this changes good. I didn't remove min and max, I add other variables to support maxMinLength.

Comment (rart) [reply static-assets/scripts/main.js:788]: Got it 👍🏻

File: ui/scss/user-dashboard/_admin.scss [hunk 4]
```diff
@@ -242,25 +242,39 @@ table {
   }
 }
 
-.user-management-dialog .popover {
-  max-width: 500px;
```
Comment (rart) [inline ui/scss/user-dashboard/_admin.scss:246]: Without this rule, the popover size forced wrapping too early

Comment (rart) [reply ui/scss/user-dashboard/_admin.scss:246]: Doesn’t look like it is anywhere else in the new set of styles

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/utils/i18n-legacy.ts [hunk 5]
```diff
@@ -154,6 +154,25 @@ const groupsAdminMessages = defineMessages({
   }
 });
 
+const settingsMessages = defineMessages({
```
Comment (rart) [inline ui/app/src/utils/i18n-legacy.ts]: Rename to profileSettingsMessages. Top var and ids.

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/utils/i18n-legacy.ts (hunk 5 above)
Comment (vestrada) [reply ui/app/src/utils/i18n-legacy.ts]: Done

File: ui/scss/user-dashboard/_admin.scss (hunk 4 above)
Comment (vestrada) [reply ui/scss/user-dashboard/_admin.scss:246]: Done

**Review by rart — APPROVED** [review]

**Pull request merged**